node_frequency: 10

net_radius: 6.25
sonar_range: 15
desired_distance: 3.0
sway_ref: 0.2
sse_threshold: 0.1
distance_of_outliers: 30
nmin_points: 100

pos_center: [30, 0]
utm_pos_inertial: [4290794.43, 491936.56]

inspection_flag: False

Vehicle: /bluerov_heavy0